crates/holochain/tests/tests/app_interface_security.rs

```diff
@@ -138,7 +138,7 @@ async fn app_interface_requires_auth() {
         .request::<_, AppResponse>(AppRequest::AppInfo)
         .await
         .unwrap_err();
-    assert_eq!("ConnectionClosed", err.to_string());
+    assert!(correct_error(&err));
```

Comment:
Yes this should be fine. As long as the connection closes, it doesn't matter exactly what message we get back. I have been wanting to look at why it varies but better to have it passing in the meantime. Thank you